Jenkinsfile

```diff
@@ -83,7 +83,6 @@ spec:
 		stage('Prepare-environment') {
 			steps {
 				container('container') {
-					sh 'mutter --replace --sm-disable &'
```

Comment:
Why did you replace `mutter` here? What do you refer to by `use tools` in the PR message?